net/mptcp/Kconfig

```diff
@@ -85,6 +85,15 @@ config MPTCP_ROUNDROBIN
 	  This is a very simple round-robin scheduler. Probably has bad performance
 	  but might be interesting for researchers.
 
+config MPTCP_REDUNDANT
+	tristate "MPTCP Redundant"
+	depends on (MPTCP=y)
```

Comment:
We probably should make this depend on MPTCP_SCHED_ADVANCED (same change is needed for MPTCP_ROUNDROBIN). But that would be a separate pull-request (as MPTCP_ROUNDROBIN needs to change as well).
Feel free to make one, if you want to.